static/elements/chromedash-feature.js

```diff
@@ -0,0 +1,492 @@
+      _receivePush: {type: Boolean, attribute: false},
+    };
+  }
+
+  constructor() {
+    super();
+    this.open = false;
+  }
+
+  // Initialize values after receiving `this.feature`.
+  firstUpdated() {
+    this._receivePush = this.feature.receivePush;
+  }
+
+  get _crBugNumber() {
+    const link = this.feature.browsers.chrome.bug;
+    if (!link) {
+      return '';
+    }
+
+    /* Get the number id from a url.
+     * Url has two formats: "http://crbug.com/111111", and
+     * "https://bugs.chromium.org/p/chromium/issues/detail?id=111111" */
+    const matches = link.match(/(id=|crbug.com\/)([0-9]+)/);
+    if (matches) {
+      return matches[2];
+    }
+    return '';
+  }
+
+  set _crBugNumber(value) {/* Never called */}
+
+  get _newBugUrl() {
+    const url = 'https://bugs.chromium.org/p/chromium/issues/entry';
+    const params = [
+      `components=${this.feature.browsers.chrome.blink_components[0] ||
+        'Blink'}`];
+    const PRE_LAUNCH_STATUSES = [
+      'No active development',
+      'Proposed',
+      'In development',
+      'Behind a flag',
+      'Origin trial',
+    ];
+    if (this._crBugNumber &&
+        PRE_LAUNCH_STATUSES.includes(
+          this.feature.browsers.chrome.status.text)) {
+      params.push(`blocking=${this._crBugNumber}`);
+    }
+    const owners = this.feature.browsers.chrome.owners;
+    if (owners && owners.length) {
+      params.push(`cc=${owners.map(encodeURIComponent)}`);
+    }
+    return `${url}?${params.join('&')}`;
+  }
+
+  set _newBugUrl(value) {/* Never called */}
+
+  get _interopRisk() {
+    if (!this.feature) return undefined;
+    const vendors = (this.feature.browsers.ff.view.val +
+                   this.feature.browsers.edge.view.val +
+                   this.feature.browsers.safari.view.val) / 3;
+    const webdevs = this.feature.browsers.webdev.view.val;
+    const standards = this.feature.standards.status.val;
+    return vendors + webdevs + standards;
+  }
+
+  set _interopRisk(value) {/* Never called */}
+
+  get _isDeprecated() {
+    const DEPRECATED_STATUSES = ['Deprecated', 'No longer pursuing'];
+    return DEPRECATED_STATUSES.includes(
+      this.feature.browsers.chrome.status.text);
+  }
+
+  set _isDeprecated(value) {/* Never called */}
+
+  get _hasDocLinks() {
+    return this.feature.resources.docs &&
+        this.feature.resources.docs.length > 0;
+  }
+
+  set _hasDocLinks(value) {/* Never called */}
+
+  get _hasSampleLinks() {
+    return this.feature.resources.samples &&
+        this.feature.resources.samples.length > 0;
+  }
+
+  set _hasSampleLinks(value) {/* Never called */}
+
+  get _commentHtml() {
+    return urlize(this.feature.comments,
+      {target: '_blank', trim: 'www', autoescape: true});
+  }
+
+  set _commentHtml(value) {/* Never called */}
+
+  _fireEvent(eventName, detail) {
+    let event = new CustomEvent(eventName, {detail});
+    this.dispatchEvent(event);
+  }
+
+  toggle(e) {
```

Comment:
What does this toggle? Can you add a comment or expand the name?

Reply:
Changed to a better name.